src/v/cluster/node/local_monitor.cc

```diff
@@ -139,5 +144,10 @@ void local_monitor::update_alert_state() {
         }
     }
 }
+void local_monitor::update_disk_metrics() {
+    auto& d = _state.disks[0];
```

Comment:
what guarantees that `_state.disks` is non-empty?

Reply:
this probably deserves to be preceded by a `vassert` that indexing into disks is valid.

Reply:
I'm assuming we'll bail out fairly soon if we have no disk available--but I like the idea of adding some asserts.